releasenotes/notes/30604.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: release-notes/v2
+kind: feature
+area: security
+issue:
+- 30603
+releaseNotes:
+- |
+  **Added** the control of XDS auth header type by XDS_TOKEN_TYPE.
```

Comment:
I don't see how this is added by this PR? doesn't the field already exist?

Reply:
Removed release notes.